regression/goto-instrument/dump-bool/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+main.c
+--dump-c
+VERIFICATION SUCCESSFUL
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
+irep
+--
+Dump-c previously always generated 0 for _Bool or __CPROVER_bool-typed
+constants. This test ensures that the value "1" is correctly generated.
```

Comment:
I don't understand how this test tests this?

Reply:
Yeah... I'm not seeing it either.  Not quite sure where the `irep` is coming from.

Reply:
Test description expanded: the previous (buggy) output made dump-c produce v = 0 instead of v = 1.